DTDL/tcads_source.json

```diff
@@ -10,23 +10,56 @@
             "name" : "AMSNetId", 
             "schema" : "string", 
             "writable" : true, 
-            "displayName" : "AMS Net Id",
-            "description" : "PLC Router Identifier for Network browsing"
+            "displayName" : "Server AMS Net Id",
+            "description" : "ADS AMS Net Id which should be connected to"
         }, 
         {
             "@type" : "Property", 
             "name" : "Port", 
             "schema" : "integer", 
             "writable" : true,
-            "displayName" : "ADS Port", 
+            "displayName" : "ADS Server Port", 
             "description" : "TwinCat ADS Port to Beckhoff PLC"
         }, 
+        {
+            "@type" : "Property", 
+            "name" : "ADSServerAddress", 
+            "schema" : "string", 
+            "writable" : true, 
+            "displayName" : "ADS Server IP Address",
+            "description" : "Whitelist Entry for Network Discovery"
+        },
+        {
+            "@type" : "Property", 
+            "name" : "RouterAMSNetId", 
+            "schema" : "string", 
+            "writable" : true, 
+            "displayName" : "Client AMS Net Id",
+            "description" : "Client Identifier for the Router Whitelist"
+        }, 
+        {
+            "@type" : "Property", 
+            "name" : "RouterPort", 
+            "schema" : "integer", 
+            "writable" : true,
+            "displayName" : "InProcess Router Port", 
+            "description" : "Exposed ADS Router Port"
+        },
         {
             "@type" : "Property",
             "name" : "PublishingInterval",
```

Comment:
@Badabunga do you think `DefaultReadCycleInterval` would be a better name?

Reply:
We can use it. The Display Name is Read Cycle Default Interval (Seconds) anyway so I will changed it.